app/views/items/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> Items </h1>
+
+<ul>
+  <% @items.each do |item| %>
+    <li>
+      Supermarket: <%= item.supermarket.name %>
+      Item: <%= item.name %>
+      Price: <%= item.price %>
+      Customers Purchased: <%= item.customers_purchased %>
```

Comment:
Good job creating a method in the Item Model to hold the logic for number of customers. Could also create a method in Item for `supermarket_name` that holds that logic. I know it's a small thing, but Law of Demeter could be applied here and the logic for getting the supermarket name could be encapsulated in a method. I like to follow the pattern of only calling one method in my view on an object, so not chaining methods to dig into objects within objects.